### ISSUE TITLE
Could a license file be added?

### ISSUE THREAD

**Issue by vermaete** [OpenDDS/opendds-monitor#46]

**Comment by jrw972** [comment]
Good catch.  We have plans to eventually convert OpenDDS to Apache2 so this would be a good pilot.  Does anyone have an objection against Apache2?

**Comment by jwillemsen** [comment]
I would prefer MIT, apache2 requires that anyone who makes local changes to document all of these explicitly (as far as I understand it)

**Comment by vermaete** [comment]
MIT?  Less to read :-)

**Comment by doug1234** [comment]
I also like MIT.

**Comment by Jonathan-L-Davis** [comment]
I would strongly prefer MIT. I think it's simplicity is a boon. It is great that it doesn't put any extra restrictions on external contributors like marking every change, or copy left.

**Comment by doug1234** [comment]
Can this be done here too:  https://github.com/OpenDDS/OpenDDW/pull/36
